PiwikTracker/Tracker.swift

```diff
@@ -186,3 +186,13 @@ extension Tracker {
         queue(event: event(withCategory: category, action: action, name: name, value: value))
     }
 }
+
+// Objective-c compatibility extension
+extension Tracker {
+    
+    @objc public func track(eventWithCategory category: String, action: String, name: String? = nil, number: NSNumber? = nil) {
```

Comment:
This here is the real fix. It is not possible to call a swift method from Objective-C when one parameter is an optional primitive.